External/Plugins/HaXeContext/Context.cs

```diff
@@ -388,6 +388,8 @@ public override void BuildClassPath()
             // add haxe libraries
             if (proj != null)
             {
+                proj.UpdateVars(false);
```

Comment:
Well then maybe we need `UpdateVars` to be called after Refresh is pressed. But it just shouldn't happen in `BuildClasspath`.